From: Bernd Zeimetz <bernd@bzed.de>
Date: Mon, 4 Oct 2010 23:46:28 +0200
Subject: [PATCH] Fixing a segfault in cgps.

As reported in <alpine.LNX.2.01.1010041235310.18045@catbert.rellim.com>
cgps segfaults when used with -u m as options. getopt did not expect
that -u takes additional options.
---
 cgps.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cgps.c b/cgps.c
index 46df222..36abc3c 100644
--- a/cgps.c
+++ b/cgps.c
@@ -777,7 +777,7 @@ int main(int argc, char *argv[])
     /*@ +observertrans @*/
 
     /* Process the options.  Print help if requested. */
-    while ((option = getopt(argc, argv, "hVl:smuD:")) != -1) {
+    while ((option = getopt(argc, argv, "hVl:smu:D:")) != -1) {
 	switch (option) {
 #ifdef CLIENTDEBUG_ENABLE
 	case 'D':
-- 
